docker/opensearch/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensearchproject/opensearch:2
+FROM opensearchproject/opensearch:3
```

Comment:
I would suggest to create separate file for opensearch2 in same way as it's done for elasticsearch8. And also add it as option in github action `elasticsearch.yml`